src/reactviews/pages/SchemaDesigner/toolbar/schemaDesignerToolbar.tsx

```diff
@@ -15,25 +15,27 @@ import { UndoRedoButtons } from "./undoRedoButton";
 
 export function SchemaDesignerToolbar() {
     return (
-        <Toolbar
-            size="small"
-            style={{
-                paddingTop: "5px",
-                paddingBottom: "5px",
-                width: "100%",
-                overflow: "hidden",
-                overflowX: "auto",
-            }}>
-            <PublishChangesDialogButton />
-            <ViewCodeDialogButton />
-            <ExportDiagramButton />
-            <ToolbarDivider />
-            <AddTableButton />
-            <UndoRedoButtons />
-            <AutoArrangeButton />
-            <DeleteNodesButton />
-            <ToolbarDivider />
-            <FilterTablesButton />
-        </Toolbar>
+        <div style={{ width: "100%", height: "30px", padding: "5px 0px" }}>
```

Comment:
Making the dimensions of the toolbar fixed. This contributed to the problems of toolbars buttons not being clickable.